src/execution_plan/ops/op_create.c

```diff
@@ -59,15 +59,15 @@ static void _CreateNodes(OpCreate *op, Record r) {
 		Node *node_ref = Record_AddNode(r, op->pending.nodes_to_create[i].node_idx, newNode);
 
 		// convert query-level properties
-		PendingProperties *converted_properties = NULL;
+		AttributeSet converted_attr = AttributeSet_New();
 		PropertyMap *map = op->pending.nodes_to_create[i].properties;
```

Comment:
wouldn't it better if we could use `op->pending.nodes_to_create[i].properties` as the attribute-set for the created node ? e.g. use `AttributeSet` instead of `PropertyMap` ?